src/command.ts

```diff
@@ -96,11 +96,12 @@ const matchImageSnapshot =
       }
 
       if (!pass && !added && !updated) {
-        const message = diffSize
-          ? `Image size (${imageDimensions.receivedWidth}x${imageDimensions.receivedHeight}) different than saved snapshot size (${imageDimensions.baselineWidth}x${imageDimensions.baselineHeight}).\nSee diff for details: ${diffOutputPath}`
-          : `Image was ${
-              diffRatio * 100
-            }% different from saved snapshot with ${diffPixelCount} different pixels.\nSee diff for details: ${diffOutputPath}`
+        const message =
+          diffSize && !options.allowSizeMismatch
```

Comment:
Is the intention here to make this something the user can pass in?

Ideally we should add it to the options type - https://github.com/simonsmith/cypress-image-snapshot/blob/master/src/types.ts#L38-L45

Reply:
The tests are failing in the pipeline, but it doesn't seem like my small change would have caused this or require a new baseline of snapshots.

I ran the tests locally on both `master` and this branch. They both succeeded with 13 passing tests.
```shell
yarn docker:build
yarn docker:run
```

Something doesn't seem right. @simonsmith can I get a sanity check here?

Reply:
Okay, if they're passing locally maybe a problem I've introduced on CI. I'll take a look for you. Thanks for checking!